src/components/Layout/Layout.js

```diff
@@ -1,17 +1,21 @@
 import React, { useEffect, useState, useRef } from "react";
 import { useSelector } from "react-redux";
+import Notification from "@canonical/react-components/dist/components/Notification/Notification";
 import PrimaryNav from "components/PrimaryNav/PrimaryNav";
 import classNames from "classnames";
 import useHover from "hooks/useHover";
 import { getViewportWidth, debounce } from "app/utils";
 
 import { isSidebarCollapsible } from "ui/selectors";
 
+import { useCookies } from "react-cookie";
+
 import "./_layout.scss";
 
 const Layout = ({ children }) => {
   const [sidebarRef, isSidebarHovered] = useHover();
   const [screenWidth, setScreenWidth] = useState(getViewportWidth());
+  const [cookies, setCookies] = useCookies(["release20_04"]);
```

Comment:
Lets just use `localstorage` then we don't need to add another dependency. You can see how to use it here: https://github.com/canonical-web-and-design/jaas-dashboard/blob/master/src/pages/Settings/Settings.js#L30

Reply:
I'm using a cookie as Safari now nukes localStorage every 7 days - we don't want this reappearing every seven days.

Reply:
I was under the impression that was only for PWA's and not general browsing. Can you find something to indicate otherwise?

Reply:
It's the opposite of that - it doesn't apply to PWAs added to the home screen but does apply to all sites accessed via Safari https://www.theregister.co.uk/2020/03/26/apple_relax_were_not_totally/

Reply:
Ahh interesting, however the kicker is this part `...if the user does not interact with the associated website during this period.` So a regular user of the application would not have their storage cleared.

It also appears that cookies have had the same limitation since Feb 19'. So I feel like my request would still be the better way forward.